src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -206,15 +248,13 @@ export class MeetingManager implements AudioVideoObserver {
     );
 
     this.audioVideo = this.meetingSession.audioVideo;
-
+    this.publishAudioVideo();
```

Comment:
Why we move `publishAudioVideo()` here. Feel like it's better to publish it after remove/add observer.

Reply:
Remove and add is different from publishing an update here. The publishAudioVideo() will just send updates that we now have a `audioVideo` object. It actually should be done as soon as we get a valid object hence moved up.

Reply:
Make sense, thanks for your explanation!